.travis.yml

```diff
@@ -41,11 +41,8 @@ env:
   - CLIPPY_VERS=0.0.187
   - RUSTFMT_VERS=0.9.0
   - DEADLINKS_VERS=0.3.0
-  - OUTDATED_VERS=0.7.0
   - SODIUM_VERS=1.0.13
   - CARGO_INCREMENTAL=1
-  - RUSTFLAGS="-C link-dead-code -D warnings"
```

Comment:
I remember that this line were added by @slowli in order to deny all warnings. Why we're removing it now?

Reply:
Nice catch: I should only remove the `link-dead-code` flag that was needed for coverage.